character.py

```diff
@@ -160,22 +160,24 @@ def __init__(self, moves):
         self.up_diagonal_air_dodge = next(ud for ud in moves if ud.name.lower() == "air dodge, diagonally up")
 
 class CharacterMiscAttributes(object):
-    def __init__(self, attributes):
-        self.weight = 0.0
-        self.gravity = 0.0
-        self.walk_speed = 0.0
-        self.run_speed = 0.0
-        self.initial_dash_speed = 0.0
-        self.air_speed = 0.0
-        self.total_air_acceleration = 0.0
-        self.short_hop_frames = 0
-        self.full_hop_frames = 0
-        self.short_hop_fast_fall_frames = 0
-        self.full_hop_fast_fall_frames = 0
-        self.fastest_out_of_shield_options = []
-        self.shield_grab_post_shield_stun = 0
-        self.shield_drop = 11
-        self.jump_squat = 3
+    def __init__(self, attributes_dict):
+        self.weight = attributes_dict["weight"]
+        self.gravity = attributes_dict["gravity"]
+        self.walk_speed = attributes_dict["walkspd"]
+        self.run_speed = attributes_dict["runspd"]
+        self.initial_dash_speed = attributes_dict["initdash"]
+        self.air_speed = attributes_dict["airspd"]
+        self.total_air_acceleration = attributes_dict["airaccel"]
+        self.fall_speed = attributes_dict["fallspd"]
+        self.fast_fall_speed = attributes_dict["fastfallspd"]
+        self.short_hop_frames = attributes_dict["shorthop"]
+        self.full_hop_frames = attributes_dict["fullhop"]
+        self.short_hop_fast_fall_frames = attributes_dict["shorthopfastfall"]
+        self.full_hop_fast_fall_frames = attributes_dict["fullhopfastfall"]
+        self.fastest_out_of_shield_options = attributes_dict["oos"]
+        self.shield_grab_post_shield_stun = attributes_dict["shieldgrab"]
+        self.shield_drop = attributes_dict["shielddrop"]
+        self.jump_squat = attributes_dict["jumpsquat"]
```

Comment:
Values like `shield drop` and `jump squat` are universal values, but the website has them anyway. Still thinking about whether or not I should hardcode this.